topology/probes/netlink/netlink.go

```diff
@@ -360,7 +361,7 @@ func newInterfaceMetricsFromNetlink(link netlink.Link) *topology.InterfaceMetric
 	}
 }
 
-func (u *NetNsProbe) updateLinkNetNsName(intf *graph.Node, link netlink.Link, metadata graph.Metadata) {
+func (u *NetNsProbe) updateLinkNetNsName(intf *graph.Node, link netlink.Link, metadata graph.Metadata) bool {
```

Comment:
look like we are not using the return value, do we really need this change?

Reply:
right I forgot something :)